### ISSUE TITLE
LaTeX: sphinxVerbatim without hllines

### ISSUE THREAD

**Issue by mgeier** [sphinx-doc/sphinx#5810]
### Problem

I'm not sure if this is really a problem, probably I'm just using it wrong ...

#### Procedure to reproduce the problem

If I'm using this in a `.rst` file:

```rst
.. raw:: latex

   \begin{sphinxVerbatim}
   abc
   \end{sphinxVerbatim}
```

#### Error logs / results

I'm getting this when running `pdflatex`:

```
! Undefined control sequence.
<recently read> \sphinx@verbatim@checkifhl 
                                           
l.107 abc
```

#### Expected results

no error, PDF with "abc" formatted as code.

If I'm using this:

```rst
.. raw:: latex

   \fvset{hllines={, ,}}%
   \begin{sphinxVerbatim}
   abc
   \end{sphinxVerbatim}
```

... everything works fine!

### Environment info
- OS: Linux, Debian testing
- Python version: 3.7.2rc1
- Sphinx version: `master`
- LaTeX: pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2019/dev/Debian) (preloaded format=pdflatex)

**Comment by jfbu** [comment]
Sorry, indeed since 1.6.6 the Sphinx LaTeX writer always sets up `hllines` key to (perhaps) emphasize lines in literal blocks, and this means naked usage of `sphinxVerbatim` environment by third party got broken. I have reimplemented it to re-establish such naked usage and will merge to 1.8 soon.

**Comment by jfbu** [comment]
Your workaround is needed for Sphinx 1.6.6 and later, and becomes unneeded but will cause no harm with (to be released) 1.8.3 and later.

**Comment by mgeier** [comment]
Thanks a lot @jfbu!
